src/test/ui/binop/binop-consume-args.stderr

```diff
@@ -261,10 +261,10 @@ LL |     drop(lhs);
    |          ^^^ value used here after move
    |
 note: calling this operator moves the left-hand side
-  --> $SRC_DIR/core/src/ops/bit.rs:LL:COL
+  --> $DIR/binop-consume-args.rs:48:5
    |
-LL |     fn bitxor(self, rhs: Rhs) -> Self::Output;
-   |               ^^^^
+LL |     lhs ^ rhs;
+   |     ^^^^^^^^^
```

Comment:
Somewhat surprised by this change, at the same time we probably can handle this better than we are (eventually)

Reply:
we may be doing some shenanigans in the diagnostics code (getting the snippet or sth like in the other case I fixed).

Reply:
ha, I was right

https://github.com/rust-lang/rust/blob/328cc90a536fdce917e701cbc04b99608637873c/compiler/rustc_borrowck/src/diagnostics/mod.rs#L1061-L1069